webrender/src/tiling.rs

```diff
@@ -515,12 +602,16 @@ impl RenderPass {
                 }
             }
 
-            render_tasks.add(&task);
-            self.targets.last_mut().unwrap().add_task(task);
+            render_tasks.add(&task, self.pass_index);
+            self.targets.last_mut().unwrap().add_task(task,
+                                                      ctx,
+                                                      render_tasks,
+                                                      self.pass_index);
         }
 
         for target in &mut self.targets {
-            target.build(ctx, render_tasks);
+            let child_pass_index = RenderPassIndex(self.pass_index.0 - 1);
```

Comment:
is this new index guaranteed to exist?
perhaps, we can have `let child_pass_index = self.pass_index.get_child().unwrap();` instead